packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -66,5 +66,6 @@
   "experimental_deploy_retrieve_description": "Enable performance enhancements to the deploy/retrieve experience.",
   "experimental_apex_library_description": "Use Apex Library for Apex CLI commands",
   "force_function_create": "SFDX: Create Function",
-  "experimental_templates_library_description": "Use Templates Library for Templates CLI commands"
+  "experimental_templates_library_description": "Use Templates Library for Templates CLI commands",
+  "functions_pull_dependencies": "Run 'npm install' to pull dependencies after creating a function"
```

Comment:
@danyusf - do you think we should say: Run npm install to pull dependencies from `package.json` after creating a function

Reply:
I guess it's usually assumed dependencies are pulled from the package.json of a Node project, although if you wanted to hedge bets on using something other than npm in the future (e.g. yarn), you could simply say "Pull dependencies from package.json"

Reply:
@danyusf - if I'm interpreting YOU and the dev guide, sounds like Node is required for JavaScript/TypeScript but something else might be required once we support additional languages?